source/changelogs/core-2025.5.markdown

```
@@ -1179,6 +1179,193 @@ For a summary in a more readable format:
+- Fix point import error ([@fredrike] - [#144462])
+- Ignore Fronius Gen24 firmware 1.35.4-1 SSL verification issue ([@farmio] - [#144463])
+- Don't encrypt or decrypt unknown files in backup archives ([@emontnemery] - [#144495])
+- Bump forecast-solar to 4.2.0 ([@bdraco] - [#144502])
+- Fix removing of smarthome templates on startup of AVM Fritz!SmartHome integration ([@mib1185] - [#144506])
+- Bump aiodns to 3.4.0 ([@bdraco] - [#144511])
+- Fix statistics coordinator subscription for lamarzocco ([@zweckj] - [#144541])
+- Update frontend to 20250509.0 ([@bramkragten] - [#144549])
+- Reolink clean device registry mac ([@starkillerOG] - [#144554])
+- Prevent errors during cleaning of connections/identifiers in device registry ([@starkillerOG] - [#144558])
+
+[#144406]: https://github.com/home-assistant/core/pull/144406
```

markdownlint-cli2 (0.17.2)

Link and image reference definitions should be needed
Unused link or image reference definition: "#144406"

(MD053, link-image-reference-definitions)